integration_test/add_friends_test.js

```diff
@@ -86,7 +84,7 @@ describe('Add friends', function() {
     // open the dropdown from the top friend request count
     await app2.client.isExisting(
       ConversationPage.oneNotificationFriendRequestTop
-    );
```

Comment:
renamed all waitForExist to isExisting last commit. But it was not checking anything then. So I need to check the return of the await there